akka-docs/src/main/paradox/general/stream/stream-design.md

```diff
@@ -45,6 +45,27 @@ All stream Processors produced by the default materialization of Akka Streams ar
 
 This means that `Sink.asPublisher(true)` (for enabling fan-out support) must be used where broadcast behavior is needed for interoperation with other Reactive Streams implementations.
 
+### Rationale and benefits from Sink/Source/Flow not directly extending Reactive Streams interfaces
+
+An sometimes overlooked crucial piece of information about [Reactive Streams](https://github.com/reactive-streams/reactive-streams-jvm/) is that they are a [Service Provider Interface](https://en.m.wikipedia.org/wiki/Service_provider_interface), as explained in depth in one of the [early discussions](https://github.com/reactive-streams/reactive-streams-jvm/pull/25) about the specification.
+Akka Streams was designed during the development of Reactive Streams, so they both heavily influenced one another.
+
+It may be sunrising to learn that even within the Reactive Specification the types had initially attempted to hide `Publisher`, `Subscriber` and the other SPI types from users of the API.
+Though since those internal SPI types would end up surfacing to end users of the standard in some cases, it was decided to [remove the API types, and only keep the SPI types][9] which are the `Publisher`, `Subscriber` et al.
+
+With this historical knowledge and context about the purpose of the standard – being an internal detail of interoperable libraries - we can with certainty say that it can't be really said that an direct _inheritance_ relationship with these types can be considered some form of advantage or meaningful differentiator between libraries.
+Rather, it could be seen that APIs which expose those SPI types to end-users are leaking internal implementation details accidentally. 
+
+The `Source`, `Sink` and `Flow` types which are part of Akka Streams, have their direct counter parts in Reactive Streams (being, respectively, `Publisher`, `Subscriber` and `Processor`), and their purpose is to provide the fluent DSL,
```

Comment:
s/counter parts/counterparts/

Reply:
I think the first comma is incorrect in this sentence
Then it might be ambigious what "their" referers to. Perhaps split in two sentences.

The `Source`, `Sink` and `Flow` types which are part of Akka Streams have the purpose to provide the fluent DSL, as well as be "factories" for running those streams. Their direct counterparts in Reactive Streams are `Publisher`, `Subscriber` and `Processor`. Being factories means that Akka Streams operate on a...

Reply:
thanks